packages/catalog-process/src/authMiddleware.ts

```diff
@@ -1,47 +1,112 @@
+/* eslint-disable functional/immutable-data */
+/* eslint-disable @typescript-eslint/naming-convention */
 import { ZodiosRouterContextRequestHandler } from "@zodios/express";
+import { logger } from "pagopa-interop-commons";
 import { match, P } from "ts-pattern";
+import { z } from "zod";
+import { AuthData } from "../../commons/src/auth/authData.js";
+import { readAuthDataFromJwtToken } from "../../commons/src/auth/jwt.js";
```

Comment:
are these imports correct? it looks like we're importing from commons via path instead of via module?